server/data/de/11k-german-sentences.txt

```diff
@@ -0,0 +1,3000 @@
+Ich habe da mal was vorbereitet.
```

Comment:
@Gregoor, Probably want to change the name of this file to be the name of the contributor (@jf99.txt in this case)

Reply:
Good point 👍 Thanks